docs/alerts/understanding-alerts.md

```diff
@@ -0,0 +1,32 @@
+# 1. Understanding Alerts in Netdata
+
+In Netdata, an **alert** (also called a "health check") is a rule that continuously monitors one or more metrics and assigns a **status** based on whether conditions are met.
+
+| Core Concept | Description |
+|--------------|-------------|
+| **Where alerts are evaluated** | Locally on each Netdata Agent or Parent node, not centrally in Netdata Cloud |
+| **How alerts work** | Each alert inspects recent metric data (for example, "average CPU over the last 5 minutes") and decides whether the system is in a healthy state (`CLEAR`), needs attention (`WARNING`), or has a problem (`CRITICAL`) |
+| **What happens on status change** | When an alert's status changes, that transition becomes an **alert event** visible in the Agent dashboard, APIs, and Netdata Cloud's Events Feed |
```

Comment:
The transition event does not become visible in the dashboard or APIs. The status is. The events *do* become visible in the Events tab.

Also they may result in notifications (depending on the configuration of notification integrations). I think this must be mentioned.

I am not sure why this mentions the *Agent* dashboard, specifically. Using this phrasing, what about Cloud?